### PR TITLE
Mention that CUSTOM_STATUS is read only

### PR DESCRIPTION
**Pull request by Andre601**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [ ] Library interface (affecting end-user code) 
- [x] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

Adds a line to the `CUSTOM_STATUS` entry mentioning, that it is read-only for bots, to prevent any possible confusion.

### CODE REVIEW

**Comment by Andre601** [comment]
Corrected some typos where `of(ActivityType, String)` mentions url and also added case where null or wrong ActivityType would cause an IllegalArgumentException

File: src/main/java/net/dv8tion/jda/api/entities/Activity.java [hunk 1]
```diff
@@ -328,6 +334,7 @@ static boolean isValidStreamingUrl(@Nullable String url)
         /**
          * Used to indicate that the {@link Activity Activity} should display as a custom status
          * in the official client.
+         * <br>This Activity type is <b>read only</b> for bots.
          *
          * @incubating This feature is currently not officially documented and might change
```
Comment (caneleex) [inline src/main/java/net/dv8tion/jda/api/entities/Activity.java:339]: now that custom status is documented and i'd say "stable", would removing the incubating annotation be okay?

Comment (Andre601) [reply src/main/java/net/dv8tion/jda/api/entities/Activity.java:339]: Do you have a Link to the documentation?

Comment (caneleex) [reply src/main/java/net/dv8tion/jda/api/entities/Activity.java:339]: https://discord.com/developers/docs/topics/gateway#activity-object-activity-types

Comment (MinnDevelopment) [reply src/main/java/net/dv8tion/jda/api/entities/Activity.java:339]: It is incubating because bots still can't use it.

**Pull request merged**